### DISCUSSION TITLE
Beginner Kotlin User (Solved)

### DISCUSSION THREAD

**Discussion by Yony55** [notKamui/Keval#27] [Q&A] [answered]
Original question

Hi,
First of all, I want to thank you for this library you coded for Kotlin string expressions.
I am trying to use it inside a project which is a calculator app. I tried to import the repository directly into "build.gradle (:app)", but it shows the following error:
Unable to resolve dependency for ':app@debugAndroidTest/compileClasspath': Could not resolve com.notkamui.libs:keval:{version}.
Maybe am doing something wrong so hope you can help me.
Thanks.

Solved
I already found my error :)
Thanks again for your contribution <3

**Comment by notKamui** [comment, marked as answer]
Hi @yony55 

> I already found my error :)
> Thanks again for your contribution <3

Haven't had the time to actually see your question. Feel free to not edit questions out next time, as these might help people who actually need the answer to said question !

Glad you could find your solution though !

btw I saw you're using Keval in your own application ! May I ask why the v0.7.2 specifically, and not the latest (v0.7.4) ?

I still recommend you to use the latest version as it has no (known) bugs at the current time, and does not break changes.

**Reply by Yony55** [reply]
Hi @notKamui 

I just edited my post to show the original question. 

As I said, I am new to this language and I didn't knew I had to replace {version} with the actual library version 😅.

Answering to your question, I googled for the library and I found the [Maven Central page](https://search.maven.org/artifact/com.notkamui.libs/keval/0.7.2/jar) for that specific version and I thought that was the final version. Thanks for letting me know, I already updated it on my project.

Thanks again for your contribution, it really helped me :)

**Reply by notKamui** [reply]
Thank you very much for your enthusiasm ! Stay tuned for the next version that will have variables support (for graphic functions for example) (which may be useful for your calculator)
